lib/mailtrap/client.rb

```
@@ -44,15 +47,72 @@ def initialize( # rubocop:disable Metrics/ParameterLists
+
+      uri = URI::HTTP.build(
+        host: api_host,
+        port: api_port,
+        path: batch_request_url
+      )
+
+      perform_request(:post, uri, {
+                        base:,
+                        requests:
+                      })
     end
```

RuboCop (1.75.5)

[convention] Method has too many lines. [14/10]

(Metrics/MethodLength)